src/apps/weblib/js-api/cssutils.js

```diff
@@ -0,0 +1,62 @@
+export const isAccessibleStylesheet = (stylesheet) => {
+  try {
+    // eslint-disable-next-line no-unused-expressions
+    stylesheet.cssRules
+    return true
+  } catch (e) {
+    return false
+  }
+}
+
+export const getCSSCustomProps = (pfx = '') =>
+  [...document.styleSheets].filter(isAccessibleStylesheet).reduce(
+    (finalArr, sheet) =>
+      finalArr.concat(
+        [...sheet.cssRules]
+          .filter((rule) => rule.constructor.name === 'CSSStyleRule')
+          .reduce((propValArr, rule) => {
+            const props = [...rule.style]
+              .filter((propName) => propName.trim().indexOf('--' + pfx + '-') === 0)
+              .map((propName) => propName.trim())
+            return [...propValArr, ...props]
+          }, [])
+      ),
+    []
+  )
+
```

Comment:
kicsit soknak érzem itt az egymásba ágyazott filtereket és reduce-okat, nehezen tudom követni

Reply:
Igen, én is. ezt a kódot csak átmozgattam.